client/src/components/ui/button.tsx

```diff
@@ -4,18 +4,23 @@ import { Slot } from '@radix-ui/react-slot';
 import { cva, type VariantProps } from 'class-variance-authority';
 
 const buttonVariants = cva(
-  'inline-flex items-center justify-center text-base ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-green-700 focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 font-sans',
+  'inline-flex items-center justify-center text-base lg:text-sm ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-green-700 focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 font-sans',
   {
     variants: {
       variant: {
         // From the UI kit
-        default: 'bg-gray-50 hover:bg-gray-150 rounded-lg',
-        primary: 'bg-green-700 hover:bg-green-800 text-white rounded-lg',
+        default: 'bg-gray-650 hover:bg-gray-600 rounded-lg focus-visible:ring-offset-0',
+        primary: 'h-10 text-white rounded-lg bg-green-700 hover:bg-green-800',
+        filters: 'h-10 rounded-lg bg-yellow-500 hover:bg-yellow-600 text-gray-700',
+        legend: 'bg-gray-700 hover:bg-gray-800 text-white',
         vanilla: '',
-        outline: 'border border-gray-300 rounded-lg px-6 py-4 hover:bg-gray-150 text-gray-700',
+        outline:
```

Comment:
This style was used on the homepage for various buttons:
- The one to access each module inside the modals
- The “Visit the platform” at the bottom of the page

The buttons used to have rounded corners and a light gray background colour on hover.

Could you fix the appearance of those buttons on the homepage? Maybe we need to create another style if you're already using this one.